package.json

```diff
@@ -32,7 +32,8 @@
     "react-markdown": "^8.0.3",
     "react-scripts": "^5.0.1",
     "semantic-ui-css": "=2.5.0",
-    "wagmi": "^0.8.6"
+    "wagmi": "^0.8.6",
+    "xss": "^1.0.14"
```

Comment:
Can you prove that we've XSS vulnerabilities?

Reply:
Actually I can't, thought it was better to prevent than remediate after it possibly happens. Will remove it.

Reply:
Actually I can't, thought it was better to prevent than remediate after it possibly happens. Will remove it.